javaparser-core-testing/src/test/java/com/github/javaparser/printer/YamlPrinterTest.java

```diff
@@ -84,7 +84,7 @@ public void testWithColonFollowedBySpaceInValue() {
     public void testWithColonFollowedByLineSeparatorInValue() {
         String expectedOutput = "---" + System.lineSeparator();
         expectedOutput += "root(Type=StringLiteralExpr): " + System.lineSeparator();
-        expectedOutput += "    value: \"a\\\\:\\\\nb\"" + System.lineSeparator();
+        expectedOutput += "    value: \"a\\\\\\\\:\\\\\\\\nb\"" + System.lineSeparator();
```

Comment:
This is from a change by @clusterrun - it seems it made these tests fail and I'm not sure if there are really supposed to be this many escape characters?!